internal/controller/setup.go

```diff
@@ -18,6 +18,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
+type docsFacade interface {
```

Comment:
minor: you may move that interface definition to `ext.go` file

Reply:
moved